cmd/server/main.go

```diff
@@ -20,9 +20,9 @@ import (
 	"net/http"
```

Comment:
"server" is possibly too generic. What about "cloud_orchestrator"?

Reply:
true, changed it to "cloud_orchestrator"